<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PiFmRds GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseFile_btn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>205</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio File Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>5</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;PiFmRds GUI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>245</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>FM Freq:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="freqData">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePathData">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setFreq_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Freq</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio File:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>540</width>
      <height>150</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'PibotoLt'; font-size:12pt; font-weight:600;&quot;&gt;Info&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>FM Freq:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ps Text:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rt Text:</string>
    </property>
   </widget>
   <widget class="QLabel" name="filePath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fmFreq">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="psText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rtText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>285</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ps Text:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="psTextData">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setPsText_btn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Ps Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setRtText_btn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Rt Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>325</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rt Text:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rtTextData">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>380</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>v0.0.1 Alpha</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>chooseFile_btn</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>freqData</zorder>
   <zorder>filePathData</zorder>
   <zorder>setFreq_btn</zorder>
   <zorder>textBrowser</zorder>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>filePath</zorder>
   <zorder>fmFreq</zorder>
   <zorder>psText</zorder>
   <zorder>rtText</zorder>
   <zorder>label_8</zorder>
   <zorder>psTextData</zorder>
   <zorder>setPsText_btn</zorder>
   <zorder>setRtText_btn</zorder>
   <zorder>label_9</zorder>
   <zorder>rtTextData</zorder>
   <zorder>start_btn</zorder>
   <zorder>version</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
